charts/vikunja/templates/deployment.yaml

```diff
@@ -0,0 +1,103 @@
+{{- if .Values.deployment.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "vikunja.fullname" . }}
+  labels:
+    {{- include "vikunja.labels" . | nindent 4 }}
+    {{- with .Values.deployment.labels }}
+    {{ toYaml . | indent 4 }}
+    {{- end }}
+  {{- with .Values.deployment.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.deployment.replicas }}
+  replicas: {{ .Values.deployment.replicas }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "vikunja.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.deployment.annotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "vikunja.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ include "vikunja.serviceAccountName" . }}
+      containers:
+        - name: {{ template "vikunja.name" . }}-api
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            {{- if .Values.ports.api.enabled }}
+            - name: api
+              containerPort: {{ .Values.ports.api.port }}
+              protocol: {{ default "TCP" .Values.ports.api.protocol | quote }}
+            {{- end }}
+          env:
+            {{- range $i, $val := .Values.env }}
+            - name: {{ $val.name | quote }}
+              value: {{ $val.value | quote }}
+            {{- end }}
+          volumeMounts:
+            - name: storage-volume
+              mountPath: {{ .Values.persistentVolumeClaim.mountPath }}
+          livenessProbe:
+            httpGet:
+              path: /
+              port: api
+          readinessProbe:
+            httpGet:
+              path: /
+              port: api
```

Comment:
Probes of API container are failing. Need to figure out correct paths. `/api/v1/health` could be worth a try?

Reply:
`/health` works fine.